### ISSUE TITLE
Checklist Data Feedback

### ISSUE THREAD

**Issue by lowlines** [Bungie-net/api#546]
I've put together some feedback after verifying/linking the checklist data to information used in my Destiny 2 Maps project.

[https://lowlidev.com.au/destiny/maps/mars](https://lowlidev.com.au/destiny/maps/mars)

**Sleeper Nodes:**
The itemHash points to the override frequency associated with it however the displayProperties don't reflect this. You can grab the description of the override frequency, which has the Rasputin code command and apply some conditional code to it to get a display name that is useful on the frontend.

Note: I'm currently applying this to all nodes which have a linked itemHash. I believe it also works for other languages.

```php
$item; // The linked item definition
$node; // The checklist definition
if (strpos($item->displayProperties->description, 'CB.NAV/RUN') !== false) {
	$node->displayProperties->name = str_replace('.', ' ', str_replace('CB.NAV/RUN.()', '', $item->displayProperties->description));
} else {
	$node->displayProperties->name = $item->displayProperties->name;
}
```
There are also several quest only sleeper nodes for the Nascent Dawn questline. These should be their own checklist since they aren't tracked by emblem stats but should still have an unobtained/obtained state based on how far along the player is in the quest. They are also per character.

- Nascent Dawn 2/5 - 1912977068
- Nascent Dawn 3/5 - 2284399300
- Nascent Dawn 4/5 - 276754608
- Nascent Dawn 5/5 - 2767578268

**Latent Memories:**
These were all over the place in terms of ordering. I put together a spreadsheet linking them up to their corresponding map id on my map, which are dynamically generated in load order (sorted by bubble then bubbleIndex). I also included extra information that could be added to the checklist definitions. It might be worth reordering these so they are grouped my bubble location!

https://docs.google.com/spreadsheets/d/1RrESSdmgmH8mlcPC5WFSsNu1KeUaxmJ1ZVKKphPdJXc/edit#gid=0

**Ghost Lore:**
I have yet to go through and verify these but offhand, there aren't any currently being tracked for the Mars destination (though none of these are tracked to the destination emblem either).

- 7 BrayTech Information Kiosks
- Warsat located in Olympus Descent and is apparently a one off scannable as part of the first mission in the campaign
- Prince Uldren's ship located in Ma'adim Subterrane, which appears to have normal ghost scan behaviour.
- Another neat thing that may or may not be possible to track are the door switches that you need to access in order to shoot some of the Latent Memories. These have a temporary state that I think resets when you leave that bubble location. If it were possible to get the state of these, this could be shown on the map.

I'll be filling out this spreadsheet over time while I verify ghost scan locations.

https://docs.google.com/spreadsheets/d/1MghOxZEEgubROzi7N3aTetJ3nA6obspLRJKg2DKzzyU/edit#gid=0

**Region Chests & Lost Sectors:**
These linked up to beautifully and required no further work on my end! ❤️

**Comment by vthornheart-bng** [comment]
Ah, good call!  Yeah, I'm just dumb counting from the first state I check onward - I'll see if I can clean that up!  Thanks!

I'll have to go see what I can dig up for those Nascent Dawn nodes... good info!

Thanks for all of this information, I appreciate it!

Some of the scannables mentioned there might not be trackable unfortunately, but I'll give it a look when I get the chance!  Thanks!

**Comment by lowlines** [comment]
Also while the information is available through the hashes, it might be more frontend friendly to include the bubble name in the displayProperties of a checklist entry so you can see which bubble location they are in at a glance (ie ghost lore).

**Comment by lowlines** [comment]
Latent Memories and Sleeper Nodes can also have the mars as the destinationHash by default since they are unique to that destination. We know this already, but it would be good to reflect this in the data also.

**Comment by vthornheart-bng** [comment]
Yeah, I dig that on both fronts.  I actually put the bubble name in the display properties initially, but then I took it out because I felt like it looked awkward/like too much data was crammed together... but I do think that utility would be useful.  I'll give it another look!

**Comment by lowlines** [comment]
It seems that faction rallies have a scannable object that spawns in several locations (mostly lost sectors) on each non-dlc destination and is per character as a one off for 5 faction tokens. The scannable disappears after your leave the bubble location.

https:\/\/www.ishtar-collective.net\/transcripts\/ghost-scan-faction-rally-io
